webpack.common.cjs

```diff
@@ -22,8 +22,8 @@ const config = {
       },
       {
         test: /\.css$/i,
-        include: path.resolve(__dirname, "src"),
```

Comment:
Hmm, the `..` should be gone. I removed this recently as I moved the location of the webpack file

Reply:
I merged branch develop into this branch to handle the conflict and perhaps it was overridden in this part.
We'll rollback it as: "include: path.resolve(__dirname, "src")", right?

Reply:
yep exactly